R/RecalFunctions.R

```diff
@@ -178,8 +179,11 @@ Recal <- function(df,
   df$mode <- mode
   names(df)[1:2] <- c("Abundance", "Exp_mass")
 
+  O_mass <- 15.9949146223
+  H_mass <- 2.01565

```

Comment:
The mass of hydrogen is for sure not 2! Should be renamed to H2_mass I assume?

Reply:
resolved in 7992e2830b8d6611823f05e6d02f750e0dab97fe